commands/blockstoragecommands/commands.go

```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "snapshot",
-			Usage:       "Used for BlockStorage Snapshot operations",
+			Usage:       "Copies of block storage volumes at a specific moment in time. Use for backups, restores, and long term storage.",
 			Subcommands: snapshotcommands.Get(),
 		},
 		{
 			Name:        "volume",
-			Usage:       "Used for BlockStorage Volume operations",
+			Usage:       "Block level volumes to expand storage on your servers.",
```

Comment:
You're right, expand could be convoluted to mean the volume itself which can't happen (IIRC).